client/src/components/context/receipt/ReceiptState.js

```diff
@@ -25,37 +25,32 @@ const ReceiptState = (props) => {
 
   // Create receipt
   const createReceipt = async (receiptData) => {
-    // const {
-    //   user_id,
-    //   title,
-    //   amount,
-    //   category,
-    //   date_created,
-    //   receipt_date,
-    //   picture_url,
-    // } = receiptData;
-    // const formData = new FormData();
-    // formData.append('user_id', user_id);
-    // formData.append('title', title);
-    // formData.append('amount', amount);
-    // formData.append('category', category);
-    // formData.append('date_created', date_created);
-    // formData.append('receipt_date', receipt_date);
-    // formData.append('picture_url', picture_url);
-    // formData.append('file_name', picture_url.name);
-
-    // for (let pair of formData.entries()) {
-    //   console.log(pair[0] + '-' + pair[1]);
-    // }
+    const {
+      user_id,
+      title,
+      amount,
+      category,
+      date_created,
+      receipt_date,
+      picture_url,
+    } = receiptData;
+    const formData = new FormData();
+    formData.append('user_id', user_id);
+    formData.append('title', title);
+    formData.append('amount', amount);
+    formData.append('category', category);
+    formData.append('date_created', date_created);
+    formData.append('receipt_date', receipt_date);
+    formData.append('picture_url', picture_url);
+    formData.append('picture_name', picture_url.name);
```

Comment:
no need to send name to backend, because `picture_url` the entire object should be sent as a binary, and backend can extract that data

Reply:
Ok. I will remove that part.